server/src/routes/api/v1/saladReviewsRouter.js

```diff
@@ -2,23 +2,25 @@ import express from 'express'
 import objection from 'objection'
 const{ ValidationError } = objection
 import { Review } from '../../../models/index.js'
+import cleanUserInput from '../../../services/cleanUserInput.js'
 
 const saladReviewsRouter = new express.Router({ mergeParams: true })
 
 saladReviewsRouter.post('/', async (req, res) => {
-    const { body } = req
+    const bodyRaw = req.body
+    const body = cleanUserInput(bodyRaw)
     const saladIdParams = req.params.id
-    const reviewerId = req.user.id
-    const formDataWithId = { ...body, saladId: saladIdParams, userId: reviewerId }
+    const reviewer = req.user
+    const formDataWithId = { ...body, saladId: saladIdParams, userId: reviewer.id }
 
     try {
         const newReview = await Review.query().insertAndFetch(formDataWithId)
+        newReview.user = reviewer
```

Comment:
if the front-end is expecting all reviews to be serialized, then this new review should also be serialized (which will appropriately include the associated user)